examples/distributed-key-value-store/README.md

```diff
@@ -0,0 +1,43 @@
+## Description
+
+This example showcases a basic distributed key-value store implemented using **libp2p**, along with the mDNS and Kademlia protocols.
+
+## Usage
+
+### Key-Value Store
+
+1.  Open two terminal windows and start two instances of the key-value store.
```

Comment:
In other examples we say specifically how to do this. Should we do that here too? Is it just `cargo run`?

Reply:
@thomaseizinger I just simplified by stating as a first step only to open two terminals (as the following talks about terminals 1 and 2, I thought it more logical to start by re-specifying to open two terminals).